components/econet/climate/__init__.py

```diff
@@ -61,6 +63,10 @@ def ensure_option_map(value):
             cv.Optional(CONF_MODE_DATAPOINT, default=""): cv.string,
             cv.Optional(CONF_CUSTOM_PRESET_DATAPOINT, default=""): cv.string,
             cv.Optional(CONF_CUSTOM_FAN_MODE_DATAPOINT, default=""): cv.string,
+            cv.Optional(
+                CONF_CUSTOM_FAN_MODE_NO_SCHEDULE_DATAPOINT, default=""
+            ): cv.string,
```

Comment:
This is weird formatting. Did some pre-commit rule force you to do this? Might be better to update/bypass the rule than have one datapoint looking all janky like this. Or pick a shorter name like *_IGNORE_SCHEDULE_*

Reply:
It's fine as is. A shorter name won't communicate this is for custom fan mode (the object is STATNFAN). And let's not update or bypass rules. Their purpose is to eliminate debates and discussions about code formatting.